applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_potential_element.cpp

```diff
@@ -118,6 +148,92 @@ namespace Kratos {
       }
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(EmbeddedCompressiblePotentialFlowElementCalculateLocalSystemRHS, CompressiblePotentialApplicationFastSuite)
+    {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateCompressibleEmbeddedElement(model_part);
+      Element::Pointer pElement = model_part.pGetElement(1);
+
+      // Define the nodal values
+      std::array<double,3> potential;
+      potential[0] = 1.0;
+      potential[1] = 2.0;
+      potential[2] = 3.0;
+
+      for (unsigned int i = 0; i < 3; i++){
+        pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL) = potential[i];
+      }
+
+      // Define the distance values
+      Vector level_set(3);
+      level_set(0) = 1.0;
+      level_set(1) = -1.0;
+      level_set(2) = -1.0;
+
+      for (unsigned int i = 0; i < 3; i++){
+        pElement->GetGeometry()[i].FastGetSolutionStepValue(GEOMETRY_DISTANCE) = level_set(i);
+      }
+
+      // Compute RHS and LHS
+      Vector RHS = ZeroVector(3);
+      Matrix LHS = ZeroMatrix(3, 3);
+
+      pElement->CalculateLocalSystem(LHS, RHS, model_part.GetProcessInfo());
+
+      std::array<double, 3> reference({0.125625, 0.0, -0.125625});
+
+      for (unsigned int i = 0; i < RHS.size(); i++) {
+        KRATOS_CHECK_NEAR(RHS(i), reference[i], 1e-6);
+      }
+    }
+
+    KRATOS_TEST_CASE_IN_SUITE(EmbeddedCompressiblePotentialFlowElementCalculateLocalSystemLHS, CompressiblePotentialApplicationFastSuite)
+    {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateCompressibleEmbeddedElement(model_part);
+      Element::Pointer pElement = model_part.pGetElement(1);
+
+      // Define the nodal values
+      std::array<double,3> potential;
```

Comment:
doesn't std::array have a list-initializer constructor?

Reply:
Yes you are right!